### DISCUSSION TITLE
Territory Administrative Hierarchy

### DISCUSSION THREAD

**Discussion by DanRunfola** [wmgeolab/geoBoundaries#2509] [General]
We are processing a large number of territory submissions (thank you @maxmalynowsky !), and it has led to an interesting challenge we're trying to figure out the best way to handle.

Taking Réunion (REU) as an example:
1) Territory of France (Island).
2) The island itself has an outline - i.e., the coastline.
3) Within Réunion, there are administrative divisons.  In France, these are ADM3 and ADM4, respectively.

Thus we have to decide how to handle three different boundaries: the country outline, ADM3 and ADM4.

For now, what I've asked the team to do is to accept three boundaries in this case: a ADM0, the outline, and then the ADM3 and ADM4.  Thus, in the case of REU, we would not have ADM1 or ADM2.  

Very much open to discussion, hence this post :).  Any thoughts are welcome.

**Comment by ibhalin** [comment]
Speaking from a french "mainlander" point of view, it feels very unintuitive that REU and other [DROM](https://en.wikipedia.org/wiki/Overseas_departments_and_regions_of_France) are not included in France's ADM0 to 2 because it's a _Région_ (but also a _Département_!) just like any mainland _Région_ minus a few dispositions.

A territory is a collection of places where the same law apply, and the law is the same in REU and in mainland France, so I don't think there's a need to exclude REU from other _Régions_. From a US point of view, it would be like not including Hawaii and Alaska in USA's ADM0 which feels wrong.

Let's take Hawaii again as an example. It's an archipelago and a territorry of the USA, it has an outline and it has administrative divisons. Should Hawaii be handle differently that any other mainland state? I don't think it should, because if it does, then what's stoping us from excluding every island from its country's mainland ?

However, a case could be made for islands that are very autonomous, like French Polynesia or maybe Guam. But autonomy is a spectrum, so we would have to define at what degree of autonomy an sub-territory should be to be excluded from its country's ADM0.
